colonel/utils/utils.py

```diff
@@ -11,23 +11,50 @@ def __init__(self, hours: int, minutes: int, seconds: int, milliseconds: int, re
         if milliseconds > 1000:
             raise BadVirtualTimeValuesError("Milliseconds should be less that 1000, " +
                                             f" but is {milliseconds}")
-        self.hours = hours
-        self.minutes = minutes
-        self.seconds = seconds
-        self.milliseconds = milliseconds
-        self.remainder = remainder
+        self.hours = int(hours)
+        self.minutes = int(minutes)
+        self.seconds = int(seconds)
+        self.milliseconds = int(milliseconds)
+        self.remainder = int(remainder)
```

Comment:
This are already int, check the types in the __init__ signature.

Reply:
The issue is that the type isn't checked in execution time so I got an error because I was getting a float, we should think how to solve this problem for all the lib